src/backend/distributed/sql/citus--9.1-1--9.2-1.sql

```diff
@@ -12,3 +12,73 @@ DROP INDEX pg_dist_colocation_configuration_index;
 CREATE INDEX pg_dist_colocation_configuration_index
 ON pg_dist_colocation USING btree(distributioncolumntype, shardcount, replicationfactor, distributioncolumncollation);
 
+CREATE TABLE citus.pg_dist_rebalance_strategy(
+    name name NOT NULL,
+    default_strategy boolean NOT NULL DEFAULT false,
+    shard_cost_function regproc NOT NULL,
+    node_capacity_function regproc NOT NULL,
+    shard_allowed_on_node_function regproc NOT NULL,
+    default_threshold float4 NOT NULL,
+    minimum_threshold float4 NOT NULL DEFAULT 0,
+    UNIQUE(name)
+);
+ALTER TABLE citus.pg_dist_rebalance_strategy SET SCHEMA pg_catalog;
+GRANT SELECT ON pg_catalog.pg_dist_rebalance_strategy TO public;
```

Comment:
@marcocitus I guess it is ok for anyone to read right? can you confirm

Reply:
seems fine, we do it for other tables